### PR TITLE
feat: add support for Kubernetes 1.13.9, 1.14.5, 1.15.2 on Azure Stack

### LINKED ISSUE #1736: Add Azure Stack VHD support for Kubernetes 1.15.2
See https://github.com/Azure/aks-engine/pull/1730

### LINKED ISSUE #1737: Add Azure Stack VHD support for Kubernetes 1.13.9
See https://github.com/Azure/aks-engine/pull/1732

### LINKED ISSUE #1738: Add Azure Stack VHD support for Kubernetes 1.14.5
See https://github.com/Azure/aks-engine/pull/1731

### PR DESCRIPTION
**Pull request by rjaini**
**Reason for Change**:
See
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.13.md#v1139
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#v1145
https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.15.md#v1152

**Issue Fixed**:

Fixes #1736 
Fixes #1737 
Fixes #1738 

**Requirements**:
- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [X] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: packer/install-dependencies.sh [hunk 1]
```diff
@@ -370,14 +370,17 @@ echo "  - busybox" >> ${RELEASE_NOTES_FILEPATH}
...
 K8S_VERSIONS="
 1.15.2
+1.15.2-azs
 1.15.1
 1.15.1-azs
 1.14.5
+1.14.5-azs
 1.14.4
 1.14.4-azs
 1.14.3
```
Comment (CecileRobertMichon) [inline packer/install-dependencies.sh]: can you please remove 1.14.3 and 1.13.7?

Comment (CecileRobertMichon) [reply packer/install-dependencies.sh]: (both the azure stack and the public cloud version)

Comment (jackfrancis) [reply packer/install-dependencies.sh]: Ah yes, thx @CecileRobertMichon 

We want to reduce any unused hyperkube image versions onto the VHD.

**Pull request merged**